src/02_evaluate_better_models.R

```diff
@@ -27,17 +27,17 @@ for(i_renglon in 1:n_renglones) {
                                              parametrosModelo = tabla_coeficientes_auxiliar)
   
   readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
-                     path =  "data/processed/parametros_modelo_logistico_laal_ig.json")
+                     path = "data/processed/parametros_modelo_logistico_laal_ig.json")
```

Comment:
Si aceptas la sugerencia de arrba, esto se vería:
```suggestion
       path = "data/processed/parametros_modelo_logistico_laal_ig.json"
    )
```